### PR TITLE
Remove AppTP

### PR DESCRIPTION
**Pull request by samsymons**
Task/Issue URL: https://app.asana.com/0/414235014887631/1206989067143299/f
Tech Design URL:
CC:

**Description**:

This PR removes AppTP.

**Steps to test this PR**:
1. Check that the app builds and runs across the board (including alpha)
2. Check that NetP connects
3. Check that you can't find any references to AppTP still hanging around in the codebase

**Device Testing**:

* [ ] iPhone SE (1st Gen)
* [ ] iPhone 8
* [ ] iPhone X
* [ ] iPhone 14 Pro
* [ ] iPad

**OS Testing**:

* [ ] iOS 14
* [ ] iOS 15
* [ ] iOS 16

---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)

### CODE REVIEW

File: Core/UserDefaultsPropertyWrapper.swift [hunk 1]
```diff
@@ -92,9 +92,6 @@ public struct UserDefaultsWrapper<T> {
 
         case shouldScheduleRulesCompilationOnAppLaunch = "com.duckduckgo.ios.shouldScheduleRulesCompilationOnAppLaunch"
 
-        case lastAppTrackingProtectionHistoryFetchTimestamp = "com.duckduckgo.ios.appTrackingProtection.lastTrackerHistoryFetchTimestamp"
-        case appTPUsed = "com.duckduckgo.ios.appTrackingProtection.appTPUsed"
```
Comment (samsymons) [inline Core/UserDefaultsPropertyWrapper.swift:96]: I haven't added logic to remove any values for these keys from user defaults as this feature never shipped, but let me know if you'd rather I handle that.

Comment (brindy) [reply Core/UserDefaultsPropertyWrapper.swift:96]: Should be fine, I think

**Comment by brindy** [comment]
@samsymons just FYI, I updated from main

**Review by brindy — CHANGES_REQUESTED** [review]
Bashed on it for a while and all seems good.  However, found a few references:
* `HomeCollectionView.configure()` (line 70) has an untyped appTP property that could be cleaned up 
* `HomeViewCollection.configure(...)` line 162, same as above
* Bunch of `appTP.` strings in Localizable - not sure when these will get removed?

**Comment by brindy** [comment]
@samsymons any updates here? :)

**Review by brindy — APPROVED** [review]
LGTM!

**Pull request merged**